src/main.rs

```diff
@@ -130,12 +132,29 @@ async fn main() -> ExitCode {
     }
 }
 
-#[instrument]
+#[instrument(name = "root")]
```

Comment:
This wasn't introduced in this PR, but by default the macro will use the function parameters as fields which is a bit verbose when it comes to the Config and Cli arguments:
```
2024-12-18T21:18:32.886193Z  INFO root{config=Config { ssh: Ssh { authorized_keys_path: ".ssh/authorized_keys", query_sshd_config: true }, hostname_provisioners: HostnameProvisioners { backends: [Hostnamectl] }, user_provisioners: UserProvisioners { backends: [Useradd] }, password_provisioners: PasswordProvisioners { backends: [Passwd] }, imds: Imds { connection_timeout_secs: 2.0, read_timeout_secs: 60.0, total_retry_timeout_secs: 300.0 }, provisioning_media: ProvisioningMedia { enable: true }, azure_proxy_agent: AzureProxyAgent { enable: true }, wireserver: Wireserver { connection_timeout_secs: 2.0, read_timeout_secs: 60.0, total_retry_timeout_secs: 1200.0 }, telemetry: Telemetry { kvp_diagnostics: true } } opts=Cli { groups: [""], config: None }}: azure_init: new
2024-12-18T21:18:32.886333Z  INFO root{config=Config { ssh: Ssh { authorized_keys_path: ".ssh/authorized_keys", query_sshd_config: true }, hostname_provisioners: HostnameProvisioners { backends: [Hostnamectl] }, user_provisioners: UserProvisioners { backends: [Useradd] }, password_provisioners: PasswordProvisioners { backends: [Passwd] }, imds: Imds { connection_timeout_secs: 2.0, read_timeout_secs: 60.0, total_retry_timeout_secs: 300.0 }, provisioning_media: ProvisioningMedia { enable: true }, azure_proxy_agent: AzureProxyAgent { enable: true }, wireserver: Wireserver { connection_timeout_secs: 2.0, read_timeout_secs: 60.0, total_retry_timeout_secs: 1200.0 }, telemetry: Telemetry { kvp_diagnostics: true } } opts=Cli { groups: [""], config: None }}: azure_init: Kernel Version: 6.12.4-200.fc41.x86_64, OS Version: 41, Azure-Init Version: 0.1.1
```

You can have it skip them both with
```suggestion
#[instrument(name = "root", skip_all)]
```